src/spacy/app.clj

```diff
@@ -34,6 +37,15 @@
         (assoc :status 302)
         (assoc-in [:headers "Location"] uri))))
 
+(defn- drop-namespace-from-keywords [event]
```

Comment:
We could theoretically modify our templates to use namespaced keywords (selmer does supports namespaces) in the future, but I think this is ok for now.